### ISSUE TITLE
How to have a callback after each step?

### ISSUE THREAD

**Issue by Pooya-Mohammadi-K** [keras-team/keras#14291]
I'm trying to schedule the learning rate after each **step** and add it as a scalar to the tensorboard. It seems the callbacks in Keras are called after each epoch ends.
Is there a way to implement it to be called after each step, or I should use the TensorFlow without

**Comment by Pooya-Mohammadi-K** [comment]
on_batch_end is the answer!!
